hugo/content/docs/learn/workflow/generate_everything.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "7. Generate your artifacts"
```

Comment:
```suggestion
title: "7. Generate the artifacts"
```

Reply:
Or even "Generate artifacts". Reason: You have not used "your" in the other captions and it looks a bit long.